rps.js

```diff
@@ -6,34 +6,38 @@ if (computerChoice < 0.34) {
 	computerChoice = "paper";
 } else {
 	computerChoice = "scissors";
-} console.log("Computer: " + computerChoice);
+} 
+
+console.log("Computer: " + computerChoice);
 
 var compare = function(choice1, choice2){
-    if (choice1 === choice2){
-        return "The result is a tie!";
```

Comment:
This is why it didn't work before, you were returning instead of using console.log.

Eventually, we'll end up converting these console.logs to returns, but for consistency, I changed it.